plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -465,6 +462,33 @@ private static Integer getIcebergFieldId(OrcColumn column)
         return Integer.valueOf(icebergId);
     }
 
+    private static Type getOrcReadType(Type columnType, TypeManager typeManager)
```

Comment:
consider making non-static so you don't have to pass `typeManager` around.

Reply:
The caller is also static, so I'd rather just leave it.